KlimaDAO/klimadao
# Fix NBO default pool project in the Frontend

## Description

Fix NBO default pool project in the Frontend
#2225 must be merged first

## Related Ticket

Closes #2091
Related to #2224

## How to Test


1. go to https://carbonmark-git-biwano-fix2091fepoolproject-klimadao.vercel.app/retire
2. Check that the default NBO project is Florestal Santa Maria Project
3. Click on this project
4. Check that the retirement targets the NBO pool (look at the url)

## Notes For QA

Specific pages, components or journeys that might be affected:
- Retire carbon from the Retire page
